lib/topological_inventory/openshift/parser/template.rb

```diff
@@ -28,7 +27,7 @@ def parse_template_notice(notice)
 
       def parse_template_tags(source_ref, tags)
         (tags || {}).each do |key, value|
-          collections[:container_template_tags].data << TopologicalInventoryIngressApiClient::ContainerTemplateTag.new(
+          collections[:container_template_tags].build(
```

Comment:
missing `collections.container_template_tags` ?

Reply:
What is the question? I only wanted demonstrate that both approaches are equal ... `collections.container_template_tags` and `collections[:container_template_tags]`

Reply:
I like more `[ ]` syntax, but the first one is more used in MiQ

Reply:
ah, I'd stick with 1 way :-)

Reply:
I'd like to prefer `collections[:xxx]` because it seems more like you are accessing some collection than using `method_missing`.

Reply:
But we aren't accessing some collection?  If we're not going to up front define the list of collections I'd rather it look like we're using method missing (since we are).

Reply:
whatever it will be, lets be consistent in the syntax :-)

Reply:
Agreed :+1:

Reply:
okay, let's don't use it explicitly, but to be consistent (even with parser_specs there), `collections[:vms]` has to work as "lazy add_collection", because now we expect it doesn't return nil.